vowpalwabbit/parse_regressor.cc

```diff
@@ -327,19 +327,21 @@ void save_load_header(
             bytes_read_write += bin_text_read_write_fixed_validated(
                 model_file, (char*)&inter_len, sizeof(inter_len), "", read, msg, text);
             if (!read)
-            {
-              memcpy(buff2, all.interactions[i].c_str(), inter_len);
+            { 
+              std::string s(all.interactions[i].begin(),all.interactions[i].end());
```

Comment:
This is a redundant copy, you can copy the buffer directly

Reply:
Changed that

Reply:
Removed